packages/itwin/property-grid/scripts/setup-tests.js

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+const fs = require("fs");
+
+const path = "../../../apps/test-viewer/.env";
```

Comment:
I'm not sure modifying the `.env` in `test-viewer` directory is the best idea. Could we set the variables straight into env? If not, maybe we could create this file somewhere under `property-grid` directory and pass the path to it to the test-viewer?

Reply:
Found a way to setup env variables in a script.

Reply:
I like your change, but I think we need to add some clarity. At the moment the script is called "setup-tests", but it not only sets up env for test viewer, but also executes given command, which runs the tests. I suggest the following:
- change the script to start the viewer by itself, without the need to pass it the command
- rename it to "start-test-viewer.js"

Reply:
Fixed